docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -164,76 +213,22 @@ java -javaagent:/path/to/opentelemetry-javaagent-all.jar \
-
-[[open-telemetry-elastic-protocol]]
-==== APM Server native support of OpenTelemetry protocol
-
-experimental::[This feature is experimental and may be changed in a future release. It is only available in a self-managed environment.]
-
-The APM Server native support of the OpenTelemetry protocol allows you to send collected telemetry data directly from your applications to APM Server.
-Trace data collected from your services and the metrics data collected from your applications and infrastructure are sent using the
-OpenTelemetry protocol.
-
-image::images/open-telemetry-protocol-arch.png[OpenTelemetry Elastic protocol architecture diagram]
-
-[float]
-[[instrument-apps-apm-server]]
-===== Instrument applications
-
-To export traces and metrics to APM Server, ensure that you have instrumented your services and applications
-with the OpenTelemetry API, SDK, or both. For example, if you are a Java developer, you need to instrument your Java app using the
-https://github.com/open-telemetry/opentelemetry-java-instrumentation[OpenTelemetry agent for Java]. 
-
-By defining the following environment variables, you can customize the OTLP endpoint so that the OpenTelemetry agent communicates with
-APM Server.
-
-[source,bash]
-----
-export OTEL_RESOURCE_ATTRIBUTES=service.name=checkoutService,service.version=1.1,deployment.environment=production
-export OTEL_EXPORTER_OTLP_ENDPOINT=https://apm_server_url:8200
-export OTEL_EXPORTER_OTLP_HEADERS="authorization=Bearer apm_secret_token"
-java -javaagent:/path/to/opentelemetry-javaagent-all.jar \
-     -classpath lib/*:classes/ \
-     com.mycompany.checkout.CheckoutServiceServer
-----
-
-|=== 
-
 | `OTEL_RESOURCE_ATTRIBUTES` | The service name to identify your application.
 
 | `OTEL_EXPORTER_OTLP_ENDPOINT` | APM Server URL. The host and port that APM Server listens for events on.
 
-| `OTEL_EXPORTER_OTLP_HEADERS` | Authorization header that includes the Elastic APM Secret token or API key: `"Authorization=ApiKey api_key"`.
-
-For information on how to format an API key, see our {apm-server-ref-v}/api-key.html[API key] docs.
-
-Please note the required space between `Bearer` and `apm_secret_token`, and `APIKey` and `api_key`.
-
-| `OTEL_EXPORTER_OTLP_CERTIFICATE` | Certificate for TLS credentials of the gRPC client. (optional)
-
 |===
 
 You are now ready to collect <<open-telemetry-elastic-traces-metrics,traces and metrics>>, <<open-telemetry-elastic-verify,verify metrics>>,
 and <<open-telemetry-elastic-kibana,visualize metrics>> in {kib}.
 
-[float]
 [[open-telemetry-elastic-traces-metrics]]
 ==== Collect traces and metrics
 
-To export traces and metrics, ensure that you have instrumented your services and applications
-with the OpenTelemetry API, SDK, or both.
-
-Here is an example of how to capture business metrics from a Java application.
+You're now ready to export traces and metrics from your services and applications.
+Here's an example of how to capture business metrics from a Java application.
```

Comment:
I have a question about this section. You can see it live, [here](https://www.elastic.co/guide/en/apm/get-started/current/open-telemetry-elastic.html#open-telemetry-elastic-traces-metrics). What's the purpose of this section? Should we be linking to the OTEL Metrics API for more information?

The original section says:
> To export traces and metrics, ensure that you have instrumented your services and applications with the OpenTelemetry API, SDK, or both.

But that same line is also said in the two instrument sections:
* [instrument 1](http://localhost:8001/guide/open-telemetry-elastic.html#instrument-apps-collector)
  > To export traces and metrics to the OpenTelemetry Collector, ensure that you have instrumented your services and applications with the OpenTelemetry API, SDK, or both. For example, if you are a Java developer, you need to instrument your Java app using the OpenTelemetry agent for Java.
* [instrument 2](http://localhost:8001/guide/open-telemetry-elastic.html#instrument-apps-apm-server)
  > To export traces and metrics to APM Server, ensure that you have instrumented your services and applications with the OpenTelemetry API, SDK, or both. For example, if you are a Java developer, you need to instrument your Java app using the OpenTelemetry agent for Java.

How we better clarify here?

Reply:
I think the purpose was to show an example that included the usage of a custom metric: `order_value`. As you say, maybe we should link to the OTEL Metrics API (if we support all the metrics now).  Or you could include an example in each *instrument* section? @cyrille-leclerc What's your thoughts on this?

Reply:
Ahh, ok, that makes sense. Added a link in 13f4961.